code/player.py

```diff
@@ -29,14 +29,17 @@
 from code import g, difficulty, task, chance
 from code import location
 from code.buyable import cash, cpu
-
+from code.stats import itself as stats, observe
 
 class DryRunInfo(object):
     pass
 
 
 class Player(object):
 
+    cash = observe("cash_earned", "_cash")
```

Comment:
Add `display=g.to_money`?

Reply:
Maybe just g.add_commas. I want the full value displayed.

Reply:
Either way works for me. :)